src/main/java/uk/gov/justice/laa/portal/landingpage/controller/UserController.java

```diff
@@ -91,4 +94,13 @@ public String displaySavedUsers(Model model) {
 
         return "users";
     }
+
+    /**
+     * Disable group of users via graph SDK
+     */
+    @PostMapping("/users/disable")
+    public String disableUsers(@RequestParam("disable-user") List<String> id) throws IOException {
+        userService.disableUsers(id);
```

Comment:
Do we need to do any auth checking here to make sure the requester has correct permissions to delete users?

Reply:
added back preauth annotation but the actual role will be decided later